charts/netbox/ci/ldap-values.yaml

```diff
@@ -0,0 +1,31 @@
+remoteAuth:
+  enabled: true
+  backends:
+    - netbox.authentication.LDAPBackend
+
+  ldap:
+    serverUri: 'ldap://ldap.forumsys.com'
+    startTls: true
+    ignoreCertErrors: false
+    caCertData: ''
+    bindDn: 'cn=read-only-admin,dc=example,dc=com'
+    bindPassword: password
+    userDnTemplate: null
+    userSearchBaseDn: 'dc=example,dc=com'
+    userSearchAttr: 'sAMAccountName'
+    groupSearchBaseDn: 'dc=example,dc=com'
+    groupSearchClass: 'group'
+    groupType: 'GroupOfNamesType'
+    requireGroupDn: 'cn=read-only-admin,dc=example,dc=com'
+      # - 'cn=read-only-admin,dc=example,dc=com'
+    findGroupPerms: true
+    mirrorGroups: true
+    mirrorGroupsExcept: null
+    cacheTimeout: 3600
+    isAdminDn: 'ou=mathematicians,dc=example,dc=com'
+      # - 'ou=mathematicians,dc=example,dc=com'
```

Comment:
this needs to be changed now that the group ldap PR is merged, right?

Reply:
Yes 🙂